src/app/components/merge/MergeContactPreview.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
This file needs to be more documented around the core logic.

Reply:
Done